pyironflow/themes.py

```diff
@@ -0,0 +1,22 @@
+from pyiron_workflow.nodes.function import Function
+from pyiron_workflow.nodes.macro import Macro
+from pyiron_workflow.nodes.transform import DataclassNode
+
+
+def node_color(node, theme):
+
+    if theme == 'light':
+        return light_mode(node)
```

Comment:
```suggestion
        return light_mode(node)
    else:
        raise ValueError(f'Theme must be one of ("light",) but got {theme}')
```

This is actually not a nit from my end -- the `if` branch needs to fail cleanly here